sdks/go/test/integration/io/xlang/bigquery/bigquery_test.go

```diff
@@ -18,14 +18,15 @@ package bigquery
 import (
 	"flag"
 	"fmt"
-	"github.com/apache/beam/sdks/v2/go/pkg/beam/register"
 	"log"
 	"math/rand"
 	"reflect"
 	"strings"
 	"testing"
 	"time"
 
+	"github.com/apache/beam/sdks/v2/go/pkg/beam/register"
```

Comment:
I think this is a consequence of running `go fmt` on the test directory (previously I ran it on `pkg/beam` only

Reply:
SGTM. It should be in the group below rather than by itself, but it's not a real problem.